model/src/main/kotlin/io/spine/validation/required/RequiredIdOptionPolicy.kt

```diff
@@ -33,13 +33,15 @@ import io.spine.protodata.ast.firstField
 import io.spine.server.event.NoReaction
 import io.spine.server.event.React
 import io.spine.server.tuple.EitherOf2
-import io.spine.validation.event.RuleAdded
+import io.spine.validation.event.RequiredFieldDiscovered
 
 /**
  * A policy that marks ID fields in entity state messages as required.
  *
- * The entity state messages are discovered via the options, specified in [ValidationConfig].
- * If ProtoData runs with no config, this policy never produces any validation rules.
+ * The entity state messages are discovered via the options, specified in
+ * [ValidationConfig][io.spine.validation.ValidationConfig].
+ *
+ * The policy does nothing if ProtoData runs without the config.
```

Comment:
I don't understand what you mean.

If ProtoData runs without a config, many policies will do nothing. Let's just kill this sentence.